js/menulis.js

```diff
@@ -0,0 +1,113 @@
+        featured_image1: "https://i.ibb.co/ZJP0mHk/rage.jpg",
+        featured_image2: "https://i.ibb.co/CMxyDMK/gray-squares.png",
+        live_link: 'https://genius.com/artists/Calvin-harris'
+    },
+
+    {
+        id: '5',
+        h5: 'Lana del Rey',
+        h6: 'Manhattan, New York, United States',
+        description: 'Elizabeth Woolridge Grant, known professionally as Lana Del Rey, is an American singer,songwriter, musician and record producer',
+        featured_image1: " https://i.ibb.co/WnxQ4rs/travis.jpg",
+        featured_image2: "https://i.ibb.co/CMxyDMK/gray-squares.png",
+        live_link: 'https://genius.com/artists/Lana-del-rey'
+    },
+
+    {
+        id: '6',
+        h5: 'Tom Yorke',
+        h6: 'Wellingborough, United Kingdom',
+        description: 'Thomas Edward Yorke is an English musician and the main vocalist and songwrit    the rock band Radiohead. A multi-instrumentalist, he mainly plays guitar and keyboards nd is known for his falsetto.',
+        featured_image1: "https://i.ibb.co/WnxQ4rs/travis.jpg",
+        featured_image2: "https://i.ibb.co/CMxyDMK/gray-squares.png",
+        live_link: 'https://genius.com/artists/Thom-yorke'
+    },
+
+]
+
+menuItems.forEach(menuItem => {
+    menuItem.addEventListener("click", toggleHambuger)
+})
+
+ham.addEventListener('click', toggleHambuger)
+menuX.addEventListener('click', toggleHambuger)
+overlay.addEventListener('click', toggleHambuger)
+
+function toggleHambuger() {
+    overlay.classList.toggle('showOverlay')
+    sideNav.classList.toggle('showNav')
+}
+
+
+
+
+const dynamicdatas = document.querySelector('.first-articles-col')
+for (let a = 0; a < speakersData.length; a++) {
+    const getHtmls = document.querySelector('.first-articles-col').innerHTML
+    const id = speakersData[a].id
+    dynamicdatas.innerHTML = `
+                                                        <article class="col-lg">
+                                                            <div class="featured-speaking-content ">
+                                                                <div class="first-articles-col row row-cols-lg-2">
+                                                                    <article class="col-lg" ${id}>
+                                                                        <a href= ${speakersData[a].live_link} target="_blank">
+                                                                            <img src= ${speakersData[a].featured_image1} alt="avatar" class="w-25 mr-3">
+                                                                                <img src=${speakersData[a].featured_image2} alt="gray-square"
+                                                                                    class="gray-square position-absolute">
+                                                                                    <div class="right-content">
```

Comment:
- [ ] Your design should fit exactly the original design provided by Cindy. Make sure that you fix that issue, take a look here below:

| Your Design | Original design |
|-------------- | --------------- |
| ![image](https://user-images.githubusercontent.com/71874425/169596621-d3bd4927-9db1-43f9-bd86-72f07da17329.png) | ![image](https://user-images.githubusercontent.com/71874425/169597600-45325a8b-4f53-49de-8b93-938fe2ec7c0b.png) |
- For this project, you should implement every single detail inspired by this original design. This will help you once you'll start working in the tech industry where you'll work with some designer who will provide you the design to implement as a software developer.